src/components/listview/listview.js

```diff
@@ -415,8 +416,9 @@ export function getListViewHtml(options) {
         }
 
         if (enableOverview && item.Overview) {
+            const overview = markdownIt().render(item.Overview || '');
```

Comment:
To be consistent with the item details, `overview` needs to be sanitized+HTML support as here: https://github.com/jellyfin/jellyfin-web/blob/d04df18ec6f4b6c8ba3d88786e804299fa97dfe9/src/controllers/itemDetails/index.js#L870

This was overlooked.

Reply:
I'll make this change.